Cargo.toml

```diff
@@ -9,3 +9,7 @@ documentation = "https://docs.rs/fixedbitset/"
 repository = "https://github.com/bluss/fixedbitset"
 
 keywords = ["container", "data-structure", "bitvec", "bitset"]
+
+[features]
+unstable = []
```

Comment:
Please don't use any crate features. Just put the benchmarks in the `benches/` directory.